src/models/rating.js

```diff
@@ -0,0 +1,42 @@
+const mongoose = require('mongoose');
+
+const { Schema } = mongoose;
+
+const raterSchema = Schema({
+  raterId: {
+    type: Schema.Types.ObjectId,
+    ref: process.env.USER_MODEL_NAME,
+    required: [true, 'Rater id is required'],
```

Comment:
Keep error messages clean in the language since frontend developers may use these on the actual UI. So here you should write ID in uppercase instead of id.